utils/_context/_scenarios/__init__.py

```diff
@@ -80,6 +80,18 @@ class _Scenarios:
         doc="Test W3C trace style",
     )
 
+    datadog_baggage_propagation = EndToEndScenario(
```

Comment:
Can we re-use the default scenario (no `DD_TRACE_PROPAGATION_STYLE` configuration) instead of adding a new scenario? We should re-use scenarios as much as possible because each scenario adds the overhead of running the agent container, weblog containers, and various test proxies